packages/compute-baseline/src/baseline/index.test.ts.snap

```diff
@@ -1,5 +1,56 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`computeBaseline disregards support that's been removed 1`] = `
+"{
+  \\"baseline\\": false,
+  \\"support\\": {}
```

Comment:
Why doesn't Safari 8 show up here? https://developer.mozilla.org/en-US/docs/Web/API/AudioTrack#browser_compatibility

Reply:
This is because the branch was behind main and the newest BCD. I've merged main into this branch and updated this snapshot.